app/src/pages/outside/loginPage/pageBlocks/serviceVersionsBlock/serviceVersionsBlock.scss

```diff
@@ -15,19 +15,18 @@
  */
 
 .service-versions-block {
-  position: absolute;
-  left: 60px;
-  bottom: 0;
-  width: 380px;
-  padding-bottom: 60px;
+  width: 100%;
+  padding: 0 50px;
+  margin-top: -40px;
```

Comment:
Did you check your changes on all breakpoints and with several `Sign In` options above the logic form?